src/ansys/dpf/core/field_definition.py

```diff
@@ -153,6 +153,61 @@ def dimensionality(self):
         self._api.csfield_definition_fill_dimensionality(self, dim, nature, dim.internal_size)
         return Dimensionality(dim.tolist(), natures(int(nature)))
 
+    def get_quantity_type(self, index=0):
+        """Getter for Quantity Type
+
+        Parameters
+        ----------
+        index: Index of the quantity type to return
+
+        Returns
+        -------
+        quantity_type : Quantity Type
+            Quantity type of the elementary data at a given index.
+        """
+        if index < 0:
+            raise ValueError("Index must be greater than or equal to 0")
+
+        quantity_type = self._api.csfield_definition_get_quantity_type(self, index)
+        return str(quantity_type)
+
+    def set_quantity_type(self, quantity_type):
+        """Setter for Quantity Type
+
+        Parameters
+        ----------
+        quantity_type: Quantity Type
+            Quantity type to set
+        """
+        self._api.csfield_definition_set_quantity_type(self, quantity_type)
+
+    def get_num_available_quantity_types(self):
```

Comment:
to be more pythonic, maybe we could have a property "num_quantity_types" ?

Reply:
@cbellot000 done